src/app/common-ui/Popup/style.scss

```diff
@@ -0,0 +1,104 @@
+.overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
```

Comment:
`left`, `right`, `width`: choose at most 2
`top`, `bottom`, `height`: choose at most 2

In this case, remove `width` and `height`.